templates/home.haml

```diff
@@ -5,7 +5,7 @@
       %p
         Memopol is reachable only in <b>reduced functionality mode</b>.
         By the way, you could access to
-        <a href="{% url 'representatives:representative-list' %}">the list of MEPs</a>.
+        <a href="{% url 'representative-list' %}">the list of MEPs</a>.
```

Comment:
As discussed on IRC those views are in memopol now. We could prefix all views with 'memopol:' but that does not seem very useful.